src/components/ContentItem/ContentItem.vue

```diff
@@ -20,10 +21,7 @@
         {{ metaText }}
       </TextParagraph>
     </div>
-    <ChevronRightIcon v-if="hasForwardButton"
-                      class="forward-icon"
-                      :data-testid="`${dataTestId}-forward-icon`"
-    />
+    <Component :is="icon" v-if="hasIcon" class="forward-icon" :data-testid="dataTestIdValue"/>
```

Comment:
(nit) please rename the `forward-icon` CSS class to something more generic (since we're now using it for both the success and forward icons). Suggestion: `item-icon`